### DISCUSSION TITLE
Bug scroll bar

### DISCUSSION THREAD

**Discussion by cameneses1** [fastapi/fastapi#8920] [Questions] [answered]
When the request body is too long fastapi does not implement a scroll bar. This causes the UX to be quite degraded.

Here is a screenshot.

![Captura web_1-7-2021_134419_ml-dev-int idboxrt com](https://user-images.githubusercontent.com/47385566/124119549-f36bfe80-da72-11eb-87c8-478235a63a68.jpeg)

**Comment by gyKa** [comment, marked as answer]
I think that is not a bug of FastAPI. It is related to [Swagger UI](https://github.com/swagger-api/swagger-ui).
